wiki/Rules/Content_Usage_Guidelines/de.md

```diff
@@ -0,0 +1,45 @@
+
+Wenn du eine Beatmap für einen Song erstellen möchtest und die Nutzungsbedingungen nicht eindeutig sind, ist es empfehlenswert, seinen Künstler nach einer Berechtigung zu fragen. Die meisten Künstler haben auf ihren Musikveröffentlichungsplattformen, persönlichen Webseiten und sozialen Medien Kontaktmöglichkeiten aufgelistet.
+
+Es mag sich wie eine große Hürde anfühlen, aber wir haben gesehen, dass Mapper recht hohen Erfolg erzielten und die Zustimmung eines Künstlers schon durch das Aufbringen des Muts erhalten haben, vor der Verwendung zu fragen. Es kann helfen, zu erwähnen, dass osu! ein kostenloses Spiel ohne Werbeanzeigen ist und du kannst sie auf unsere [Lizenzierungsseite](/wiki/Legal/Music_licensing) verweisen, um ihnen eine bessere Vorstellung davon zu vermitteln, wie Inhalte konsumiert werden.
+
+Während Musik häufig urheberrechtlich geschützt ist, stehen Soundeffekte für benutzerdefinierte Hitsounds öfters kostenlos zur Verfügung. Zum Beispiel bietet das Philharmonia Orchester kostenlose [Hörbeispiele von Instrumenten](https://philharmonia.co.uk/resources/sound-samples/) an und [Freesound](https://freesound.org) erlaubt die Suche nach Hörbeispielen, entsprechend gefiltert nach Nutzungsrechten.
+
+## Visuell
+
+Visuelle Inhalte beinhalten den [Hintergrund](/wiki/Beatmap/Background)/das [Storyboard](/wiki/Storyboard)/Video einer Beatmap und folgen denselben Anforderungen für die Nutzung wie Audioinhalte.
+
+Wie bei Musik empfehlen wir, nach den Nutzungsangaben des Künstlers zu suchen und ihn nach einer Erlaubnis zu fragen, wenn diese unklar sind. Falls du beispielsweise daran interessiert bist, einen Hintergrund eines Künstlers auf pixiv zu verwenden, mache deren Profil ausfindig für die Nutzungsbedingungen oder sende ihm eine Nachricht, um nach einer Erlaubnis zu fragen.
+
+Viele Webseiten für Bild-Hosting haben Suchfilter für Lizenzierung nach Creative Commons oder nicht-kommerzielle Verwendung. Einige Webseiten, die Mapper häufig benutzen, um urheberrechtsfreie Hintergründe zu finden, sind unter anderem:
+
+- [pixabay](https://pixabay.com/)
+- [Unsplash](https://unsplash.com/)
+- [Pexels](https://www.pexels.com/)
+
+Wenn du die Erlaubnis eines Künstlers erhalten hast, ihren Song, ihr Album-Cover oder ein anderes offizielles Kunstwerk benutzen zu dürfen, kann es voraussichtlich auch als Hintergrund/Storyboard verwendet werden. Aber auch hier gilt, frage nach einer Erlaubnis, wenn das nicht eindeutig ist.
+
+## Gameplay
+
+Beim Einreichen einer Beatmap solltest du eine Erlaubnis für Inhalte bezogen auf das Gameplay haben, wie zum Beispiel [Gast-Schwierigkeitsgrade](/wiki/Beatmap/Guest_difficulty) oder Gameplay von anderen Rhythmusspielen.
+
+Wir lehnen keine Einreichungen mit Gameplay (Hit-Objekte) ab, das dem aus anderen Spielen ähnelt, da dies nicht respektvoll gegenüber der  Rhythmusspiel-Community ist.
```

Comment:
```suggestion
Wir lehnen Einreichungen mit Gameplay (Hit-Objekte) ab, das dem aus anderen Spielen ähnelt, da dies nicht respektvoll gegenüber der Rhythmusspiel-Community ist.
```

Reply:
I must have read the sentence in an incorrect way. Maybe `dulden keine` sounds better here in this case or is closer to the original meaning?

Reply:
Yeah that works better actually, I updated my suggestion.